omf/fileio/geoh5.py

```diff
@@ -585,7 +585,7 @@ def collect_attributes(
         """
         with fetch_h5_handle(workspace):
             if isinstance(element, UidModel):
-                values = np.r_[getattr(element, "array")]
+                values = element.array.array
```

Comment:
guess array.array is legit there? a bit strange

Reply:
OMF already stores it as a numpy array. That np.r_ conversion was slowing down a ton, going from a non-numpy to np.ndarray. OMF already stores it as a numpy array, so no need to convert.